openedx/features/course_experience/templates/course_experience/course-home-fragment.html

```diff
@@ -31,6 +31,11 @@
           id="upsell-modal",
           props={},
         )}
+        ${static.renderReact(
```

Comment:
Instead of using ENABLE_DASHBOARD_TAB, we should probably use the new flag.  We should do the same for all modals.

Additionally, does it make sense to just re-use UpsellExperimentModal for all experiments and update from Optimizely?  I assume that was @MatthewPiatetsky's plan for the 3rd experiment, but maybe verify?

Reply:
I was going to use optimizely because it only has a single slide instead of 4. I would need to use optimizely to hide yo elements like the next button anyway

Reply:
Makes sense.  Thanks @MatthewPiatetsky.

Reply:
@jaebradley: If you are leaving your own modal, just wrap it with its own new waffle flag as you noted.

Reply:
@robrap which one is "the new flag"?